### PR TITLE
dovi_tool: 2.0.3 (new formula)

### PR DESCRIPTION
**Pull request by Kreeblah**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/d/dovitool.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  head "https://github.com/quietvoid/dovi_tool.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "CLI tool combining multiple utilities for working with Dolby Vision",
```
Comment (chenrui333) [inline Formula/d/dovitool.rb:16]: need a better test to test the tool functionality

Comment (Kreeblah) [reply Formula/d/dovitool.rb:20]: I'm attempting to write another test here, but I'm having trouble accessing the files extracted from the source code archive.  There's a file in there with a relative path of `assets/tests/fel_orig.bin` that I'm trying to reference, and I keep getting a missing file error.  I thought I'd be able to use `#{buildpath}` to reference it, but that doesn't seem to have content in a test block, and the contents of the source archive don't appear to be available in `#{testpath}`.

What would be the best way of referencing this file?

File: Formula/d/dovitool.rb
Comment (chenrui333) [inline Formula/d/dovitool.rb:1]: file name should be devi-tool or devi_tool to match the artifact.

File: Formula/d/dovitool.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class Dovitool < Formula
+  desc "CLI tool for Dolby Vision metadata on video streams"
+  homepage "https://github.com/quietvoid/dovi_tool/"
+  url "https://github.com/quietvoid/dovi_tool/archive/refs/tags/2.0.3.tar.gz"
+  sha256 "6a4afe2733ab1a4eca6ff98c6038f60bbbed5c649b99c239759ccf0b17a3f818"
+  license "MIT"
+  head "https://github.com/quietvoid/dovi_tool.git", branch: "main"
+
+  depends_on "rust" => :build
+
```
Comment (chenrui333) [inline Formula/d/dovitool.rb:10]:
```suggestion

  on_linux do
    depends_on "fontconfig"
  end
```

**Review by chenrui333 — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**